plugins/contact.py

```diff
@@ -0,0 +1,101 @@
+""" A bot to make the wordkeeper's job a little easier during games of Contact.
+
+wordkeeper> !challenge @player1 @player2
+slaskbot> @player1 @player2: type !ready when here
+player1> !ready
+player2> !ready
+slaskbot> (@wordkeeper) Starting countdown...
```

Comment:
Do slaskbot's own messages trigger the onMessage hook? That's what I assumed when I built this.